std-bits/src/main/java/org/enso/image/opencv/OpenCV.java

```diff
@@ -0,0 +1,480 @@
+// Copied from https://github.com/openpnp/opencv/blob/v4.5.1-0/src/main/java/nu/pattern/OpenCV.java
+// to allow library loading on GraalVM.
```

Comment:
Is the file copied without any changes? This comment should say whether it's modified or not. Does the license let us do this?

Reply:
Yes, we can use the file, it's under the BSD license. The license comment was added during the legal review.